Makefile

```diff
@@ -187,6 +187,8 @@ ifeq ($(detected_OS),Darwin)
   endif
 endif
 
+IPFS_GATEWAY_URL_ARG := https://ipfs.status.im/
+PROD_IPFS_GATEWAY_URL_ARG := https://status-im.infura-ipfs.io/ipfs/
```

Comment:
Are you sure you want to use `:=` and make it not overridable?

Reply:
If you want env variables to override those use lazy assignment: `?=`

Also, it seems to me like `IPFS_GATEWAY_URL_ARG` is not actually used since it's always replaced by `PROD_IPFS_GATEWAY_URL_ARG`. Wouldn't it make more sense and call those `IPFS_GATEWAY_URL_ARG_DEV` and `IPFS_GATEWAY_URL_ARG_PROD`?